packages/terra-i18n-plugin/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file is odd. Basically things processed by webpack we don't want modules compiled, but things not processed by webpack (e.g. plugins imported into webpack config) need modules compiled because they are executed by node, not webpack.